providers/mailjet/src/lib/mailjet.provider.ts

```diff
@@ -44,6 +44,11 @@ export class MailjetEmailProvider implements IEmailProvider {
           Subject: options.subject,
           TextPart: options.text,
           HTMLPart: options.html,
+          Attachments: options.attachments.map((attachment) => ({
+            ContentType: attachment.mime,
+            Filename: attachment.name,
+            Base64Content: attachment.file.toString('base64'),
```

Comment:
@devblin Haven't heard about `toString('base64')` did you had the change to test it in real to see if it's sending the email with the attachments?

Reply:
Hey, so `toString('base64')` is to encode file contents in base64 string which is required as per mailjet attachment doc.
And, I have tested the provider with file attachments (txt, pdf, gif files), and its sending files flawlessly 💯

Reply:
You are awesome! :)